docusaurus/README.md

```diff
@@ -0,0 +1,33 @@
+# Website
+
+This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+
+## Installation
+
+```console
+yarn install
+```
+
+## Local Development
+
+```console
+yarn start
```

Comment:
Am I missing something or some docusaurus files are missing from the project files?

Reply:
Some files were missing